### PR TITLE
[pkg-cfize] Finish integrating Cloud Foundry exporter into release

### PR DESCRIPTION
**Pull request by fnichol**
See the commit message comments for more details. Small things, but each is broken out for clarity and for any possible future `git revert` if necessary.

tenor-256274910

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by fnichol** [comment]
Just found one tweak I'd like to make… will repush…

File: .travis.yml [hunk 1]
```diff
@@ -202,8 +202,8 @@ matrix:
     - os: linux
       env:
         # These components will build as Habitat packages in the provided order
-        - COMPONENTS="hab-butterfly hab plan-build backline bintray-publish studio pkg-aci pkg-export-docker pkg-mesosize pkg-tarize sup"
-        - AFFECTED_DIRS="support/ci/deploy\.sh|Cargo\.toml|Cargo\.lock|VERSION|$_RUST_HAB_BIN_COMPONENTS|$_RUST_HAB_LIB_COMPONENTS"
+        - COMPONENTS="hab-butterfly hab plan-build backline bintray-publish studio pkg-aci pkg-dockerize pkg-cfize pkg-export-docker pkg-mesosize pkg-tarize sup"
```
Comment (reset) [inline .travis.yml]: Could you also update `.bldr.toml` with any directory triggers for this component?

Comment (fnichol) [reply .travis.yml]: Yep, I'll add that in here

**Comment by fnichol** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/284457087).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/284457087).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**